pandas/core/indexes/accessors.py

```diff
@@ -324,7 +321,12 @@ def __new__(cls, data):
 
         orig = data if is_categorical_dtype(data) else None
         if orig is not None:
-            data = Series(orig.values.categories, name=orig.name, copy=False)
+            data = Series(
+                orig.values,
```

Comment:
make this orig.array (doesn't make a practical difference), but is more idiomatic to our usecase here.